recipe/meta.yaml

```diff
@@ -11,7 +11,6 @@ source:
 
 build:
   number: 0
-  skip: true   # [win]
```

Comment:
@AliKhoda you still have not put the vc features here.
Please do so according this guide: https://conda-forge.org/docs/buildwin.html#vc-features